powerquery-docs/connectors/automy-data-analytics.md

```diff
@@ -1,22 +1,22 @@
 ---
 title: Power Query Automy Data Analytics Connector
 description: Provides basic information and prerequisites for the Automy Data Analytics connector
-author: DougKlopfenstein
+author: Carlos Quintero
 ms.topic: conceptual
-ms.date: 1/8/2024
-ms.author: dougklo
+ms.date: 11/25/2024
+ms.author: cquintero89
 ---
 
-# Automy Data Analytics (Beta)
+# Automy Data Analytics
 
 > [!NOTE]
->The following connector article is provided by ACEROYALTY, the owner of this connector and a member of the Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or have changes you would like to see made to this article, visit the ACEROYALTY website and use the support channels there.
+>The following connector article is provided by AUTOMY, the owner of this connector and a member of the Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or have changes you would like to see made to this article, visit the AUTOMY website and use the support channels there.
 
 ## Summary
 
 | Item | Description |
 | ---- | ----------- |
-| Release State | Beta |
+| Release State | Production |
```

Comment:
For consistency with all of the other connector articles, please change "Production" to "General Availability".